### ISSUE TITLE
Importing with webpack 5 causes an error

### ISSUE THREAD

**Issue by Adjective-Object** [ai/nanoid#352]
enhanced-resolve (used by webpack) now enforces that "default" be the last export in an exports map.
https://github.com/webpack/enhanced-resolve/blob/02d99a2e6852adaf43fecfc3fed14d8d5c8df10b/lib/util/entrypoints.js#L455

in nanoid 3.3.1, the "." export's 'types' field follows its "default" field. It should precede the "default" field

```jsonc
{
  "name": "nanoid",
  "version": "3.3.1",
  //...
  "exports": {
    ".": {
      "browser": "./index.browser.js",
      "require": "./index.cjs",
      "import": "./index.js",
      "default": "./index.js",
      "types": "./index.d.ts"
    },
    "./async": {
      "browser": "./async/index.browser.js",
      "require": "./async/index.cjs",
      "import": "./async/index.js",
      "default": "./async/index.js"
    }
    // ..
}.

**Comment by ai** [comment]
Thanks. Fixed https://github.com/ai/dual-publish/commit/651bba39ba955f20c68f6c23d163f6a58b5e9b33

**Comment by ai** [comment]
The fix was released in Nano ID 3.3.2.
